rcl/src/rcl/event.c

```diff
@@ -121,7 +123,7 @@ rcl_subscription_event_init(
   }
 
   return rmw_subscription_event_init(&event->impl->rmw_handle,
```

Comment:
Should reformat this line similar to how you formatted the `return rmw_publisher_event_init()` line.

Reply:
Will do